### DISCUSSION TITLE
How to create sub-module ?

### DISCUSSION THREAD

**Discussion by Danyihza** [nWidart/laravel-modules#1546] [Q&A]
I just want to separate modules for their scope, I want to create Auth for User and Admin then I want to separate them into two folders inside Modules. Is that possible to create with artisan command?

Example : 
Modules -> 
  - User
    - Auth
      - Config
      - Console
      - Database
      - etc
  - Admin
    - Auth
      - Config
      - Console
      - Database
      - etc

**Comment by dcblogdev** [comment]
sorry, this is now supported in this package.

what you can do instead is turn on scan mode in the config and add multiple paths that can be scanned, you would have to update the namespaces for all files within those modules placed in the defined paths.

```
'scan' => [
        'enabled' => true,
        'paths' => [
            base_path('OtherModules/*'),
            base_path('vendor/*/*'),
        ],
    ],
```

**Reply by bamilawal** [reply]
How do use artisan command for these other scan paths?
